partition_gpu/partition_gpu.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This daemonset deploys the GPU partitioner on all GPU nodes and partitions
+# the GPUs as defined in the GPU config file.
+
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
this eventually will become an init container of the device plugin daemonset right?

Reply:
Yup, that's the plan. For now, since this can be used independently, I created a daemonset for it.